pkg/common/tracer/stats/event.go

```diff
@@ -88,8 +89,8 @@ var (
 
 // errors
 var (
-	ErrNotAllowed = errors.New("event definition is not allowed after initialization")
-	ErrDuplicated = errors.New("event name is already defined")
+	errNotAllowed = errors.NewPublic("event definition is not allowed after initialization")
+	errDuplicated = errors.NewPublic("event name is already defined")
```

Comment:
Will there be users who have used this public variable in the project? Will changing to private variable cause a break change.

Reply:
Change to public, thx!